bin/runtests

```diff
@@ -399,11 +411,19 @@ function init_bundle($phpunitPath)
     }
 
     write_info('Doctrine ORM detected, updating the schema for the bundle.');
+
+    if ($input->getOption('database-update-drop')) {
+        exec_cmd(sprintf('%s %s', $console, 'doctrine:database:drop --force'));
+        exec_cmd(sprintf('%s %s', $console, 'doctrine:database:create'));
+    }
```

Comment:
This is not required now since we do not use foreign keys with Sqlite. Remove what is not required ?